workspaces/lightspeed/plugins/lightspeed/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "@red-hat-developer-hub/backstage-plugin-lightspeed",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
Not sure if you need to manually update the version if you're adding changesets.

Reply:
I believe the changeset handles this in a subsequent PR. See https://github.com/redhat-developer/rhdh-plugins/actions/workflows/version-bump.yml

cc: @04kash

Reply:
Yup, no need to manually update the version. Merging this PR should generate a Version Packages PR similar to this one: https://github.com/redhat-developer/rhdh-plugins/pull/64

Reply:
Reverted the change.